<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GridFaceClass</class>
 <widget class="QMainWindow" name="GridFaceClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>627</width>
    <height>451</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GridFace</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="btn_run">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>10</y>
      <width>281</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>RUN FOR THE HILLS</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_msg">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>190</y>
      <width>521</width>
      <height>151</height>
     </rect>
    </property>
    <property name="text">
     <string>Message</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_browser">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>80</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Browser</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>627</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="gridface.qrc"/>
 </resources>
 <connections/>
</ui>
